### DISCUSSION TITLE
Transfer Syquel/mise-android-sdk plugin

### DISCUSSION THREAD

**Discussion by Syquel** [mise-plugins/.github#16] [Transfer Request]
I already requested to transfer the [Syquel/mise-android-sdk](https://github.com/Syquel/mise-android-sdk) plugin to @jdx .

**Comment by dynamsoft-h** [comment]
Hello @jdx, is this plugin able to be transferred, and hence added to the `mise` registry? It does not work with `ubi`; is it preferable to add it to the `aqua` registry instead of the `mise` registry? Thanks! CC @Syquel

**Reply by jdx** [reply]
sorry, it's expired, can you send it again?

**Reply by Syquel** [reply]
I sent the transfer request again

**Comment by Syquel** [comment]
@jdx Any chance to get some more permissions in the new repo at [mise-plugins/mise-android-sdk](https://github.com/mise-plugins/mise-android-sdk/ ) so I can maintain the plugin?

**Reply by jdx** [reply]
no, sorry. I don't permit anyone except myself and the advisory board to create commits for supply chain security reasons. It's not that I think you wouldn't be trustworthy, this is a promise I've already made to the users.
